api-report/map.api.md

```diff
@@ -131,19 +137,36 @@ export interface IDirectoryValueChanged extends IValueChanged {
 
 // @public
 export interface ILocalValue {
-    makeSerialized(serializer: IFluidSerializer, bind: IFluidHandle): ISerializedValue;
+    // @alpha
+    makeSerialized(serializer: IFluidSerializer, bind: IFluidHandle, attribution?: AttributionKey): ISerializedValue;
```

Comment:
we can't mark an existing API alpha. I'm not sure if we have great precedent for this tbh. This isn't part of the typical API surface though, so I think leaving a note in the constructor that passing defined values for `attribution` is in alpha is ok.